src/scvi/external/gimvi/_module.py

```diff
@@ -80,11 +78,11 @@ def __init__(
         self,
         dim_input_list: list[int],
         total_genes: int,
-        indices_mappings: list[Union[np.ndarray, slice]],
+        indices_mappings: list[np.ndarray | slice],
         gene_likelihoods: list[str],
         model_library_bools: list[bool],
-        library_log_means: list[Optional[np.ndarray]],
-        library_log_vars: list[Optional[np.ndarray]],
+        library_log_means: list[np.ndarray | None],
```

Comment:
is this one correct? Does it expect a list with None

Reply:
This is what was written before. I think that because optional is inside the list it turned it into an option of a list of Nones to a list of np.ndarray